src/jit/hwintrinsicxarch.cpp

```diff
@@ -186,6 +186,33 @@ bool Compiler::isIntrinsicAnIsSupportedPropertyGetter(NamedIntrinsic intrinsic)
     }
 }
 
+// TODO - remove the fully implemented ISA
```

Comment:
I think that all these `TODO` comments should be omitted. I expect that it will take some time to complete the current set of ISAs, and there are likely to be more in future.